File: set_item.cs
// Random C# function generated on 2025-06-23
using System;
using System.Collections.Generic;
using System.Linq;

public class GetValue
{
    public int Id { get; set; }
    public string Name { get; set; }
    public double Value { get; set; }
    public DateTime CreatedAt { get; set; }
    
    public GetValue(int id, string name, double value)
    {
        Id = id;
        Name = name;
        Value = value;
        CreatedAt = DateTime.Now;
    }
}

public static class Create_valueProcessor
{
    public static List<GetValue> Create_value(IEnumerable<GetValue> items)
    {
        const double multiplier = 3.0;
        
        return items
            .Where(item => item.Value > 0)
            .Select(item => new GetValue(
                item.Id,
                $"{item.Name}_processed",
                item.Value * multiplier))
            .ToList();
    }
    
    public static void Main()
    {
        var testData = new List<GetValue>
        {
            new GetValue(1, "Item1", 23),
            new GetValue(2, "Item2", 71)
        };
        
        var result = Create_value(testData);
        Console.WriteLine($"Processed {result.Count} items");
    }
}

File: set_item.cpp
// Random C++ function generated on 2025-09-16
#include <iostream>
#include <vector>
#include <string>
#include <algorithm>

struct Item {
    int id;
    std::string name;
    double value;
    
    Item(int id, const std::string& name, double value) 
        : id(id), name(name), value(value) {}
};

std::vector<Item> update_response(const std::vector<Item>& items) {
    const double multiplier = 4.0;
    std::vector<Item> result;
    
    std::transform(items.begin(), items.end(), std::back_inserter(result),
        [multiplier](const Item& item) {
            return Item(item.id, item.name + "_processed", item.value * multiplier);
        });
    
    return result;
}

int main() {
    std::vector<Item> testData = {
        Item(1, "item1", 36.0),
        Item(2, "item2", 78.0)
    };
    
    auto result = update_response(testData);
    std::cout << "Processed " << result.size() << " items" << std::endl;
    
    return 0;
}
